samples/bluetooth/mesh/light/README.rst

```diff
@@ -22,15 +22,14 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820, thingy53_nrf5340_cpuapp
 
-.. note::
-   When used with :ref:`zephyr:thingy53_nrf5340`, the sample supports the MCUboot bootloader with serial recovery and SMP DFU over Bluetooth.
-   Debug logs are provided over the USB CDC ACM class serial port.
-
 The sample also requires a smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
 
   * `nRF Mesh mobile app for Android`_
   * `nRF Mesh mobile app for iOS`_
 
+.. note::
+   |thingy53_sample_note|
+
```

Comment:
This aplay for both the light and light_switch sample: When doing dfu upgrade by the mobile app, from light to light_switch sample or vice a versa. it is important that the device is not provisioned. If the sample is provisioned the sample will not be possible to provision after dfu upgrade and it is not possible to reset the old provisioned device. So it is important to reset the provisioned device (that way unprovision it) before doing a dfu upgrade by the mobil app.
I wrote this here, as I do not know the best way to put this in the doc. This aplay for both the light and ligth_switch sample.

Reply:
Hi @Balaklaka 
Thanks a lot for your input. Isn't the provisioning data handled in Zephyr's settings storage? In that case it should be by default automatically cleared when DFU is performed. In general we cannot assume that settings will be compatible between various samples or applications, so settings partition should be erased before booting a new application.

Reply:
I do not necessary think that it is correct to automatically delete the settings when DFU is performed, because if we update to a new version of the same example we want to keep settings.

Reply:
I think deleting settings by default would be the safest option (if we have no additional information about the firmware that is being swapped). Otherwise we may encounter strange issues.
If I recall correctly, settings cleanup is done using separate mcumgr command, so we may consider introducing additional logic that would prevent deleting settings if this is not necessary (e.g. if we are sure that settings remain compatible).

Reply:
We had a discussion on this, and it is a to complex fix now. And most likely it has to be fixed upstream. So my suggestion for this release is to add a comment about this for now.

Reply:
Added short description related to settings erase

Reply:
I will discuss with Wayland and MAG about making the settings erase optional. Enabled by default, but a user can retain the settings partition by un-checking a box or similar.